kolibri/plugins/user/assets/src/modules/profile/actions.js

```diff
@@ -40,20 +42,14 @@ export function updateUserProfile(store, { edits, session }) {
       store.commit('SET_PROFILE_ERROR', { isError: false });
     },
     error => {
-      const { status, entity } = error;
-      let errorMessage = '';
-      if (status.code === 400) {
-        errorMessage = Object.values(entity)[0][0];
-      } else if (status.code === 403) {
-        errorMessage = entity[0];
+      const errorsCaught = CatchErrors(error, [ERROR_CONSTANTS.USERNAME_ALREADY_EXISTS]);
+      if (errorsCaught) {
+        store.commit('SET_PROFILE_ERRORS', errorsCaught);
```

Comment:
Getting `unknown local mutation type: SET_PROFILE_ERROR, global type: profile/SET_PROFILE_ERROR`

Reply:
That must be coming from line 42. Should that be a different mutation?

Reply:
SET_PROFILE_ERROR**S** (MIssing the S)